src/theme/login/register.ftl

```diff
@@ -1,23 +1,59 @@
-                        <h2>DoD PKI User Registration</h2>
-                        <p>
-                            ${cacSubjectDN}
-                        </p>
+                    <div class="row">
+                        <div class="col-lg-12">
+                            <div class="alert alert-info">
+                                <div class="row">
+                                    <div class="col-lg-1 align-items-start d-flex col-alert-icon">
+                                        <img src="${url.resourcesPath}/img/icon_information.png" />
+                                    </div>
+                                    <div class="col">
+                                        <h3>DoD PKI User Registration</h3>
+                                        <p>${cacSubjectDN}</p>
+                                    </div>
+                                </div>
+                            </div>
+                        </div>
                     </div>
                 <#else>
                     <#if properties["USERNAME_PASSWORD_AUTH_ENABLED"] == "false" && properties["X509_LOGIN_ENABLED"] == "true">
-                        <div class="alert alert-warning">
-                            <h2>Registration Requirement</h2>
-                            <p>No CAC detected. A DoD PKI CAC is required to complete registration.</p>
+                        <div class="row">
+                            <div class="col-lg-12">
+                                <div class="alert alert-warning">
+                                    <div class="row">
+                                        <div class="col-lg-1 align-items-start d-flex col-alert-icon">
+                                            <img src="${url.resourcesPath}/img/icon_warning.png" />
+                                        </div>
+                                        <div class="col">
+                                            <h3>Registration Requirement</h3>
+                                            </p>No CAC detected. A DoD PKI CAC is required to complete registration.</p>
+                                        </div>
+                                    </div>
+                                </div>
+                            </div>
                         </div>
                     </#if>
                 </#if>
+
+                <div class="row">
+                    <div class="col-lg-12">
+                        <div class="alert alert-info">
+                            <div class="row">
+                                <div class="col-lg-1 d-flex align-items-center col-alert-icon">
+                                    <img src="${url.resourcesPath}/img/icon_information.png"  />
+                                </div>
+                                <div class="col">
+                                    <p>Use your company or government email address as your access will be based off of your validated email address.</p>
```

Comment:
More of a question - is this "universally" true/something we want in the default theme? By default I don't think we automatically map permissions based on email which is why I ask. Even for our Google setup, permissions mapping is done based on a mapper in Google.

Reply:
Adjusted according to the internal conversation